activesupport/lib/active_support/array_inquirer.rb

```diff
@@ -9,8 +9,10 @@ module ActiveSupport
   #   variants.desktop?  # => false
   class ArrayInquirer < Array
     # Passes each element of +candidates+ collection to ArrayInquirer collection.
-    # The method returns true if at least one element is the same. If +candidates+
-    # collection is not given, method returns true.
+    # The method returns true if any element from the +candidates+ collection
+    # matches stringified or symbolized form of any element in the ArrayInquirer collection.
```

Comment:
https://github.com/rails/rails/pull/24974/files/e619a940e72f826fdfbf6bf16ff7a767168474dd#r62857123 is causing confusion. We don't stringify elements from ArrayInquirer collection, but the candidate themselves.